sdks.txt

```diff
@@ -0,0 +1,259 @@
+Loading package information...                                                  
Loading local repository...                                                     
[=========                              ] 25% Loading local repository...       
[=========                              ] 25% Fetch remote repository...        
[=========                              ] 25% Fetch remote repository...        
[==============                         ] 36% Fetch remote repository...        
[=================                      ] 43% Fetch remote repository...        
[=================                      ] 44% Fetch remote repository...        
[==================                     ] 46% Fetch remote repository...        
[==================                     ] 47% Fetch remote repository...        
[==================                     ] 47% Fetch remote repository...        
[===================                    ] 48% Fetch remote repository...        
[===================                    ] 49% Fetch remote repository...        
[===================                    ] 50% Fetch remote repository...        
[====================                   ] 50% Fetch remote repository...        
[====================                   ] 51% Fetch remote repository...        
[====================                   ] 52% Fetch remote repository...        
[=====================                  ] 53% Fetch remote repository...        
[=====================                  ] 53% Fetch remote repository...        
[=====================                  ] 54% Fetch remote repository...        
[=====================                  ] 55% Fetch remote repository...        
[======================                 ] 56% Fetch remote repository...        
[======================                 ] 56% Fetch remote repository...        
[======================                 ] 57% Fetch remote repository...        
[=======================                ] 58% Fetch remote repository...        
[=======================                ] 59% Fetch remote repository...        
[=======================                ] 59% Fetch remote repository...        
[========================               ] 60% Fetch remote repository...        
[========================               ] 60% Computing updates...              
[=============================          ] 75% Computing updates...              
[=============================          ] 75% Computing updates...              
[=======================================] 100% Computing updates...             
Installed packages:

```

Comment:
What's this file for?

Reply:
Just me debugging something.  I'll remove in a separate PR.

Reply:
I'm going to fix it here.